activesupport/lib/active_support/railtie.rb

```diff
@@ -13,6 +13,13 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
+    initializer "active_support.halt_callback_chains_on_return_false", after: :load_config_initializers do |app|
```

Comment:
@claudiob this caused a subtle change in our initializing order https://github.com/troter/rails/commit/50f51b0ded5958c8104cd739a4220cc82d822df8.

@matthewd yeah. I had the same suspicion, but for what I could understand this is how our TSort implementation is working. I'll investigate better today, but I fell that if we change this we may have more problems.